query/select.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/influxdata/influxql"
 )
 
+var DefaultTypeMapper = influxql.MultiTypeMapper(
```

Comment:
this will allocate a `TypeMapper` interface{} wrapper each time:

```go
valuer := influxql.TypeValuerEval{
 		TypeMapper: DefaultTypeMapper,
 	}
```

However, you can avoid that allocation if you change the declaration to:

```go
var DefaultTypeMapper TypeMapper = influxql.MultiTypeMapper(
```

Reply:
Done.